plenum/common/messages/client_request.py

```diff
@@ -65,6 +67,14 @@ class ClientGetTxnOperation(MessageValidator):
     )
 
 
+class ClientTxnAuthorAgreementOperation(MessageValidator):
+    schema = (
+        (TXN_TYPE, ConstantField(TXN_AUTHOR_AGREEMENT)),
+        (TXN_AUTHOR_AGREEMENT_TEXT, LimitedLengthStringField(max_length=TXN_AUTHOR_AGREEMENT_TEXT_SIZE_LIMIT)),
+        (TXN_AUTHOR_AGREEMENT_VERSION, LimitedLengthStringField(max_length=TXN_AUTHOR_AGREEMENT_VERSION_SIZE_LIMIT))
+    )
```

Comment:
I think it makes sense to add simple tests to verify that schema like here `https://github.com/hyperledger/indy-plenum/blob/63fb89a41e7799a1f7c6cac3dc4a6654ee45f690/plenum/test/input_validation/message_validation/test_client_message.py`

Reply:
Will fix in next PR

Reply:
Done